### ISSUE TITLE
Bug: NordVPN Variable SERVER_REGIONS dont work anymore

### ISSUE THREAD

**Issue by nonameoio** [qdm12/gluetun#1711]
### Is this urgent?

Yes

### CPU arch

x86_64

### VPN service provider

NordVPN

### What are you using to run the container

Portainer

### What is the version of Gluetun

latest

### What's the problem 🤔

Hello, since NordVPN change her login policy, i have an issue with setting the SERVER_REGIONS variable. 
Anyone have the same issue?

### Share your logs

```plain text
2023-06-29T09:40:28Z INFO [vpn] starting
2023-06-29T09:40:28Z INFO [firewall] allowing VPN connection...
2023-06-29T09:40:28Z INFO [openvpn] OpenVPN 2.5.8 x86_64-alpine-linux-musl [SSL (OpenSSL)] [LZO] [LZ4] [EPOLL] [MH/PKTINFO] [AEAD] built on Nov  2 2022
2023-06-29T09:40:28Z INFO [openvpn] library versions: OpenSSL 3.1.1 30 May 2023, LZO 2.10
2023-06-29T09:40:28Z INFO [openvpn] TCP/UDP: Preserving recently used remote address: [AF_INET]XXX
2023-06-29T09:40:28Z INFO [openvpn] UDP link local: (not bound)
2023-06-29T09:40:28Z INFO [openvpn] UDP link remote: [AF_INET]XXX
2023-06-29T09:40:29Z INFO [openvpn] [hk274.nordvpn.com] Peer Connection Initiated with [AF_INET]185.225.234.19:1194
2023-06-29T09:40:31Z INFO [openvpn] TUN/TAP device tun0 opened
2023-06-29T09:40:31Z INFO [openvpn] /sbin/ip link set dev tun0 up mtu 1500
2023-06-29T09:40:31Z INFO [openvpn] /sbin/ip link set dev tun0 up
2023-06-29T09:40:31Z INFO [openvpn] /sbin/ip addr add dev tun0 XXX
2023-06-29T09:40:31Z INFO [openvpn] UID set to nonrootuser
2023-06-29T09:40:31Z INFO [openvpn] Initialization Sequence Completed
2023-06-29T09:40:33Z INFO [healthcheck] healthy!
```

**Comment by Xanthey** [comment]
Getting AUTH_FAILED when it's trying to connect. Verified credentials, repulled container, tried VPN connection through another app, worked fine. Waiting for update

**Comment by Cedrik12** [comment]
> 

Are you using Nordvpn ? If yes, see this discussion: https://github.com/qdm12/gluetun/discussions/1710

**Comment by qdm12** [comment]
> Hello, since NordVPN change her login policy, i have an issue with setting the SERVER_REGIONS variable.

I don't see any error in your logs

```
2023-06-29T09:40:31Z INFO [openvpn] Initialization Sequence Completed
2023-06-29T09:40:33Z INFO [healthcheck] healthy!
```

I guess you're having the auth error as does @Xanthey so see closed issue #1708 it's also mentioned on the nordvpn wiki page now.

Closing this for now since it's resolved / user error, feel free to re-open an issue if this is something else.

**Comment by Xanthey** [comment]
> > Hello, since NordVPN change her login policy, i have an issue with setting the SERVER_REGIONS variable.
> 
> I don't see any error in your logs
> 
> ```
> 2023-06-29T09:40:31Z INFO [openvpn] Initialization Sequence Completed
> 2023-06-29T09:40:33Z INFO [healthcheck] healthy!
> ```
> 
> I guess you're having the auth error as does @Xanthey so see closed issue #1708 it's also mentioned on the nordvpn wiki page now.
> 
> Closing this for now since it's resolved / user error, feel free to re-open an issue if this is something else.
> 

I fixed it with the same emailed instructions. It was on Nord's side, not GT
